src/pages/failure/index.tsx

```diff
@@ -35,21 +35,18 @@ export const FailurePage = ({ finishOfframping, continueFailedFlow, transactionI
             </p>
             <p>Either try to continue or start over.</p>
           </>
-        ) : (
-          <p className="mt-6 text-center">
-            Unfortunately, your withdrawal request could not be processed. Please try again.
-          </p>
-        )}
+        ): (<p></p>)}
         {failure === 'recoverable' && (
           <button className="w-full mt-5 text-white bg-blue-700 btn rounded-xl" onClick={continueFailedFlow}>
             Continue
           </button>
         )}
         <button className="w-full mt-5 text-white bg-blue-700 btn rounded-xl" onClick={finishOfframping}>
-          Start over
+          Try again
         </button>
         <div className="h-0.5 m-auto w-1/5 bg-pink-500 mt-8 mb-5" />
-        <p className="text-center text-gray-400">If you continue to experience issues, contact support on:</p>
+        <p className="text-center text-gray-400">In case you experience any issues, please copy this {transactionId && <TransactionInfo transactionId={transactionId} />} </p>
+        <p className="text-center text-gray-400">Contact our support team at: We’re here to help!</p>
```

Comment:
Maybe we should rename split this differently somehow? I feel it would make more sense to have 'Contact our support team at:' Then show the Telegram button and then below say 'We're here to help!'. 
Or rephrase to 'Contact our support team on Telegram. We're here to help!' and then show the button as is.

Reply:
So for telegram i dont know how to do [this](https://github.com/pendulum-chain/vortex/pull/196#issuecomment-2396893938), that is how it should be